### PR TITLE
[FUSEQE-5449] Batch Update automated test for DB connector

### PR DESCRIPTION
**Pull request by tplevko**
//test: `@webhook-db-batch`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert 
Integration - Webhook to DB | Webhook to db with batch insert 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: ui-tests/src/test/resources/features/integrations/webhook-db.feature [hunk 1]
```diff
@@ -24,6 +24,7 @@ Feature: Integration - Webhook to DB
 
     When select the "Webhook" connection
     And select "Incoming Webhook" integration action
+    And sleep for "1000" ms
```
Comment (avano) [inline ui-tests/src/test/resources/features/integrations/webhook-db.feature]: why?

File: ui-tests/src/test/resources/features/integrations/webhook-db.feature [hunk 2]
```diff
@@ -67,3 +68,51 @@ Feature: Integration - Webhook to DB
...
+  Scenario: Webhook to db with batch insert
+
+    When navigate to the "Home" page
+    And click on the "Create Integration" link
+    Then check visibility of visual integration editor
+
+    And check that position of connection to fill is "Start"
+    When select the "Webhook" connection
+    And select "Incoming Webhook" integration action
+    And sleep for "1000" ms
```
Comment (avano) [inline ui-tests/src/test/resources/features/integrations/webhook-db.feature]: why?

File: ui-tests/src/test/resources/features/integrations/webhook-db.feature [hunk 3]
```diff
@@ -67,3 +68,51 @@ Feature: Integration - Webhook to DB
...
+    And select "Invoke SQL" integration action
+    And fill in invoke query input with "INSERT INTO TODO(task, completed) VALUES(:#task, :#completed)" value
+    And select "Yes" from "batch" dropdown
+    And click on the "Done" button
+
+    When add integration step on position "0"
+    And select the "Data Mapper" connection
+    And open data mapper collection mappings
+    Then check visibility of data mapper ui
+    And open data mapper collection mappings
```
Comment (avano) [inline ui-tests/src/test/resources/features/integrations/webhook-db.feature]: when

File: ui-tests/src/test/resources/features/integrations/webhook-db.feature [hunk 4]
```diff
@@ -67,3 +68,51 @@ Feature: Integration - Webhook to DB
...
+      | task      | task      |
+      | completed | completed |
+
+    And click on the "Done" button
+    And publish integration
+    And set integration name "DB batch Insert"
+    And publish integration
+    Then wait until integration "DB batch Insert" gets into "Running" state
+
+    And select the "DB batch Insert" integration
```
Comment (avano) [inline ui-tests/src/test/resources/features/integrations/webhook-db.feature]: when

File: ui-tests/src/test/resources/features/integrations/webhook-db.feature [hunk 5]
```diff
@@ -67,3 +68,51 @@ Feature: Integration - Webhook to DB
...
+    And invoke post request to webhook in integration webhook-to-db with token test-webhook and body [{"task": "write doc", "completed": 1},{"task": "publish doc", "completed": 0}]
+
+    And check that query "SELECT * FROM TODO WHERE task = 'write doc'" has some output
```
Comment (avano) [inline ui-tests/src/test/resources/features/integrations/webhook-db.feature]: then

**Comment by tplevko** [comment]
This is currently blocked due to: https://issues.jboss.org/browse/ENTESB-11328

**Comment by tplevko** [comment]
Closing for now, will reopen once it's fixed

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request closed without merging**